clang/lib/Serialization/ASTReader.cpp

```diff
@@ -9403,6 +9404,20 @@ DiagnosticBuilder ASTReader::Diag(SourceLocation Loc, unsigned DiagID) const {
   return Diags.Report(Loc, DiagID);
 }
 
+void ASTReader::warnStackExhausted(SourceLocation Loc) {
+  // When Sema is available, avoid duplicate errors. This should be rare.
```

Comment:
```suggestion
  // When Sema is available, avoid duplicate errors.
```

Reply:
Good catch, I don't think this would necessarily hold in practice in the next few years.